.github/workflows/docs.yml

```diff
@@ -0,0 +1,13 @@
+name: docs
+
+on: [pull_request]
+
+jobs:
+  generate-docs:
+    name: Doxygen
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mattnotmitt/doxygen-action@1.9.8
```

Comment:
The version (`1.9.8`) does not match the version defined in the Doxyfile (`1.9.7`).
It causes `doxygen -u .doxygen/Doxyfile` warning.

Reply:
Updated the Doxyfile to 1.9.8 version.